vhdbuilder/release-notes/AKSWindows/2019-containerd/17763.4851.230914-image-list.json

```diff
@@ -0,0 +1,268 @@
+﻿{
+    "sku":  "2019-containerd",
+    "imageVersion":  "17763.4851.230905",
+    "imageBom":  [
+                     {
+                         "id":  "sha256:afb8056ee409207ffcd8f6dcde16efe7893ee5706ffb4a09fad58608096046b8",

```

Comment:
seems the formatting is a bit off?

Reply:
Thanks for the comment. I reformat it in vs code with "alt + shift+ f". I think I need to find to solution in code way later.